mcs/class/Mono.Profiler.Log/Mono.Profiler.Log/LogEnums.cs

```diff
@@ -199,4 +201,22 @@ public enum LogSynchronizationPoint {
 		WorldStop = 1,
 		WorldStart = 2,
 	}
+
+	public enum HeapRootKind {
```

Comment:
A few things:

* Please add a comment above the enum that points to the C enum (as for the others here).
* I prefer to use explicit enum values when we're mapping to a C enum, especially if that C enum is using explicit values too.
* I think we should use singular names here, i.e. `StaticVariable`, `GCHandle`, etc.
* `Marshalling` -> `Marshal` for consistency with the rest of .NET.